ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj

```diff
@@ -389,11 +385,7 @@
 					"$(inherited)",
 					_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION,
 				);
-				OTHER_LDFLAGS = (
-					"$(inherited)",
-					"-Wl",
-					"-ld_classic",
-				);
+				OTHER_LDFLAGS = "$(inherited)";
```

Comment:
Similar to what I [share here](https://github.com/WordPress/gutenberg/pull/59461#pullrequestreview-1908779207), not sure why this is changing 🤔

Reply:
Following the conversation [here](https://github.com/WordPress/gutenberg/pull/59461#discussion_r1507679647), these changes were being reverted as my local command line tools were not using Xcode 15. I've updated that now, and confirmed these changes are no longer happening when running `bundle exec pod install --repo-update`. I've reverted these changes in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6687/commits/29272135494d34e63df1e198fa3474acd1ec9659.